cypress/integration/pet_applications_spec.js

```diff
@@ -0,0 +1,20 @@
+describe('Pet Applications Component', () => {
```

Comment:
Great job writing these tests before merging to see whether it is giving us what we want it to.